mudlet-mapper.xml

```diff
@@ -4132,7 +4171,7 @@ mmp.settings:setOption(matches[3], val)</script>
 				<packageName></packageName>
 				<regex>^area list$</regex>
 			</Alias>
-			<AliasGroup isActive="no" isFolder="yes">
+			<AliasGroup isActive="yes" isFolder="yes">
```

Comment:
We don't want to have mapping commands enabled on update:
```suggestion
			<AliasGroup isActive="no" isFolder="yes">
```

Reply:
Whoops! That was oversight on my part.